applications/API/admin.py

```
@@ -0,0 +1,142 @@
+        "id",
+        "content",
+        "category",
+        "reports_count",
+        "status",
+        "view_reports_link",
+    )
+    inlines = [InsultReviewInline]
+    search_fields = ("content", "category__name", "status")
+    list_filter = ("category", "status", "nsfw", "added_by", HasPendingReviewFilter)
+
+    def save_model(self, request, obj, form, change):
```

Ruff (0.12.2)

Missing return type annotation for public function `save_model`

Add return type annotation: `None`

(ANN201)